content/00.front-matter.md

```diff
@@ -25,12 +30,31 @@ on {{manubot.date}}.
 
 ## Authors
 
+### Version 2.0 Authors
```

Comment:
Will you add the other authors from https://github.com/greenelab/deep-review/issues/959 in a follow up pull request?  Or in this pull request once we agree on the formatting?

Reply:
Yea - my goal at this point was to just get the formatting down.